scripts/ci/gke.sh

```diff
@@ -275,6 +277,53 @@ teardown_gke_cluster() {
     gcloud container clusters delete "$CLUSTER_NAME" --async
 
     info "Cluster deleting asynchronously"
+
+    create_log_explorer_links
+}
+
+create_log_explorer_links() {
+    if [[ -z "${ARTIFACT_DIR:-}" ]]; then
+        info "No place for artifacts, skipping explorer links"
+        return
+    fi
+
+    artifact_file="$ARTIFACT_DIR/gke-logs-summary.html"
+
+    cat > "$artifact_file" <<- HEAD
+<html style="background: #fff">
+    <head>
+        <title><h4>GKE Logs Explorer</h4></title>
+    </head>
+    <body>
+    <ul style="padding-bottom: 28px; padding-left: 30px; font-family: Roboto,Helvetica,Arial,sans-serif;">
+HEAD
+
+    local start_ts
+    start_ts="$(cat /tmp/GKE_CLUSTER_CREATED_TIMESTAMP)"
+    local end_ts
+    end_ts="$(date -u +"%Y-%m-%dT%H:%M:%SZ")"
+    local project
+    project="$(gcloud config get project --quiet)"
+
+    for authUser in {0..2}; do
+        echo \
+\<li\>\
+\<a href=\"https://console.cloud.google.com/logs/query\;query=\
+resource.type=%22k8s_container%22%0A\
+resource.labels.cluster_name%3D%22"$CLUSTER_NAME"%22%0A\
+resource.labels.namespace_name%3D%22stackrox%22%0A\
+\;timeRange="$start_ts"%2F"$end_ts"\
+\;cursorTimestamp="$start_ts"\
+?authuser="$authUser"\
+\&project="$project"\
+\&orgonly=true\&supportedpurview=organizationId\"\>authUser "$authUser"\</a\>\
```

Comment:
```suggestion
\&orgonly=true\&supportedpurview=organizationId\" target=\"_blank\" \>authUser "$authUser"\</a\>\
```

Reply:
Unfortunately `target=_blank` does not work in an iframe (at least in chrome). It opens a new tab with a 'content blocked' message. Maybe it needs a 'right click and open in new tab message/label' but that seems overly silly IMO.